### PR TITLE
Bump monitoring setup manifests

### PR DESCRIPTION
**Pull request by fabxc**
@s-urbaniak

### CODE REVIEW

File: modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1alpha1
```
Comment (s-urbaniak) [inline modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml]: Do you mind to change the corresponding line in https://github.com/coreos/tectonic-installer/blob/1b4c579/modules/tectonic/resources/tectonic.sh#L130?

Just replace:
```
kubectl create -f monitoring/prometheus-k8s.json
```
with
```
kubectl create -f monitoring/prometheus-k8s.yaml
```

File: modules/tectonic/resources/manifests/updater/app-version-tectonic-monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: coreos.com/v1
```
Comment (s-urbaniak) [inline modules/tectonic/resources/manifests/updater/app-version-tectonic-monitoring.yaml]: The same comment as above applies here regarding https://github.com/coreos/tectonic-installer/blob/1b4c579/modules/tectonic/resources/tectonic.sh#L167.

Comment (fabxc) [inline modules/tectonic/resources/manifests/updater/app-version-tectonic-monitoring.yaml]: done

File: modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml (hunk 1 above)
Comment (fabxc) [inline modules/tectonic/resources/manifests/monitoring/prometheus-k8s.yaml]: done

**Comment by s-urbaniak** [comment]
@fabxc generally looks good and deploys successfully, but I get no data:

i.e.
```
https://<url>/api/kubernetes/api/v1/proxy/namespaces/tectonic-system/services/prometheus:9090/api/v1/query_range?query=pod_name%3Acontainer_memory_usage_bytes%3Asum%7Bpod_name%3D%27default-http-backend-1363258968-g2dvs%27%7D&start=1492787305.436&end=1492790905.436&step=30
```
yields
```
{"status":"success","data":{"resultType":"matrix","result":[]}}
```

This is both for pods and nodes. Do you have any tips on further debugging?

**Comment by s-urbaniak** [comment]
After testing and discussing OOB with @fabxc this needs some more rework.

**Comment by fabxc** [comment]
To add more context:

Our TPO e2e tests run against a cluster installed with tectonic-installer and passed. Manual verification was looking good too.

Problems encountered:
* node-exporter daemon set did not deploy to master nodes
* recording rules where configured correctly but not loaded
* superseded manifest not removed in this PR yet

No user-facing changes in this iteration, so this does not hurt being deferred to the next release.

**Comment by chancez** [comment]
Closing, as the other PR has merged.

**Pull request closed without merging**